onnxruntime/test/platform/apple/apple_package_test/apple_package_test.xcodeproj/project.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Workspace
+   version = "1.0">
+   <FileRef
+      location = "self:/Users/rachguo/Repos/onnxruntime11/onnxruntime/onnxruntime/test/platform/apple/apple_package_test/apple_package_test.xcodeproj">
```

Comment:
this path is specific to a machine. does this file need to be checked in?

Reply:
removed! thanks for the catch up.